IBAnimatable/TransitionFromDirection.swift

```diff
@@ -25,6 +27,8 @@ public enum TransitionFromDirection {
     case .Bottom:
       // The actual transition direction is oposite, need to reverse
       return kCATransitionFromTop
+    default:
+      return ""
```

Comment:
I was thinking `fatalError`, but I think returning `kCATransitionFromLeft` as the default value is not a bad idea, how do you think @tbaranes ?